docs/ROADMAP.md

```diff
@@ -0,0 +1,86 @@
+
+**Target Date:** Tuesday, March 24, 2020
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
+
+Technical Features / Requirements:
+
+* App built to be submitted on Apple App Store and Google Play Store, including code signing process
+* UI styled to exact [design spec in Figma](https://www.figma.com/file/fjzTIloCEK4FpbyDiTLj2X/iOS-UX) ([#186](https://github.com/WorldHealthOrganization/app/issues/186))
+  * Displaying static content in English only
+* App is still functional in an offline environment with default content able to be accessed without a network connection ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* Demonstrate capability to localize content from the technical platform perspective, even if other languages/cultures are _NOT_ part of that v0.1 launch
+  * Framework in place to localize UI elements ([#126](https://github.com/WorldHealthOrganization/app/pull/126) :white_check_mark:)
+  * Framework in place to load the informational content in localized formats ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* Demonstrate capability to load content dynamically from the cloud, even if this feature will _NOT_ be turned on as part of the v0.1 launch ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* _(Open Question)_ Demonstrate capability to instrument the app and collect basic analytics around which features and pages are actually being used, even if this feature will _NOT_ be turned on as part of the v0.1 launch ([#183](https://github.com/WorldHealthOrganization/app/issues/183))
+  * Investigating legal requirements of this data collection, including whether updated TOS/Privacy Policy is needed
+
+Features _NOT_ included in v0.1:
+
+* Localization of initial content across languages/cultures outside of English / US
+  * Blockers: translation & adaptation of content to enough other languages/cultures to be a meaningful initial feature
+  * We are building in the capabilities into v0.1 from the technical perspective
+* Triage survey telling users whether or not they should go to the hospital
+  * Blockers: full definition on what the survey entails + legal review on the implications of making medical suggestions
+  * _Question:_ do we want to link to an existing triage survey in a webview for v0.1, and if so, which one?
+
+Non-Technical Requirements for v0.1:
+
+* All content & assets required for submission to Apple App Store and Google Play Store ([#188](https://github.com/WorldHealthOrganization/app/issues/188), [#187](https://github.com/WorldHealthOrganization/app/issues/187) :warning:)
+* Confirmed informational content & organization of that content included in app ([#93](https://github.com/WorldHealthOrganization/app/issues/93) :warning:)
+* Any legal requirements for submission to app stores & legal review ([#17](https://github.com/WorldHealthOrganization/app/issues/17) :warning:)
```

Comment:
I'm concerned that :warning: is being used too much. Should be used sparingly
```suggestion
* Any legal requirements for submission to app stores & legal review #17 :warning:
```

Reply:
Understood - I'll remove unless I see it as a potential show-stopper for hitting the milestone